### ISSUE TITLE
Staging Review finding: Focus lost after clicking cancel.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#81367]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Authenticated Experience - Profile
**Product:** Profile
**Feature:** CH33 EDU Direct Deposit

## Findings details
**VA.gov Experience Standard - issue:** [User can't determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When editing your direct deposit information in the "has EDU" scenario (test user 380), when clicking Cancel to abandon that process keyboard focus doesn't appear to go anywhere. Pressing [tab] gets you somewhere that makes sense, so it's not a problem for most keyboard users. But for screen reader users, an element receiving focus determines what's announced next by the screen reader. Since focus isn't moved anywhere, the screen reader doesn't announce anything --- leaving users to guess whether the button worked or not.

<details><summary>Link, screenshot or steps to recreate</summary>

I use the [What has focus? bookmarklet](https://codepen.io/svinkle/pen/WgYRxq) to track focus through the page in the console, but any focus tracker should work here.

Steps to reproduce:

1. Navigate to [/profile/direct-deposit](https://staging.va.gov/profile/direct-deposit) as test user 380 user with direct deposit information.
2. Activate a screen reader (eg. VoiceOver on Mac, NVDA on Windows for easy testing).
3. Press [tab] to move focus to Edit.
4. Press [enter].
5. Press [tab] until focus moves to the Cancel button.
6. Press [enter]. Note that nothing is announced by your screen reader. In the console, nothing is logged as receiving focus.
7. Press [tab] to move focus back to the Edit button.
8. Press [enter].
9. Select checking, then press [tab] until focus moves to the Cancel button.
10. Press [enter].
11. In the confirmation modal, press [tab] to move focus to the Yes button.
12. Press [enter]. Note that nothing is announced by your screen reader. In the console, nothing is logged as receiving focus.
13. Press [tab] to move focus back to the Edit button.
14. Press [enter].
15. Select checking, then press [tab] until focus moves to the Cancel button.
16. Press [enter].
17. In the confirmation modal, press [tab] to move focus to the No button.
18. Press [enter]. Note that nothing is announced by your screen reader. In the console, nothing is logged as receiving focus.

</details>

## Recommended action

This behavior only appears to be happening in the "has EDU" scenario. Focus behaves as expected in the other scenarios. Not sure what's happening there, but hopefully it's an easy bug to track down. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by adamwhitlock1** [comment]
This focus issue has been fixed and is available on staging to validate. Here are some quick gifs of using mouse and keyboard to cancel editing.

<img width="1014" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeVdoQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f6f8bb7b1d988a1d66b57ec5fcf92c25f9ff802d/mouse.gif" alt="mouse.gif" />

<img width="1014" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeWFoQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--209c7757248848e8b563edac3f8e55adfa20e8bf/keyboard.gif" alt="keyboard.gif" />

**Comment by shiragoodman** [comment]
@adamwhitlock1 are you asking for help from Governance team to validate? CC @briandeconinck

**Comment by adamwhitlock1** [comment]
Yep! @briandeconinck would you be able to validate this for me?

**Comment by briandeconinck** [comment]
@adamwhitlock1 Sorry for the delay, and thank you @mtcA6 for the nudge!

Tested, verified focus behaves as expected, everything looks good to me!
